Lib/Environment.cpp

```diff
@@ -174,36 +163,7 @@ bool Environment::haveOutput()
 ostream& Environment::out()
 {
   ASS(_outputDepth);
```

Comment:
All of these `env.out` related functions are now almost trivial. Could they go altogether in a subsequent PR?

Reply:
Yes, let's remove `env.out`. It's not been used consistently throughout anyway (and was doing the same as `std::cout <<`, I think).